### ISSUE TITLE
CMS #25663 "Can't build admin menu item" error while (re)installing component

### ISSUE THREAD

**Issue by pjwiseman** [joomla/joomla-platform#1441]
This CMS issue has been held up for a while due to it being a "platform" change without triggering the platform change process.  It's a workaround to fix a problem introduced by a failed installation.  It's been tested twice in CMS.   There was subsequently a minor platform change (#1338 mbabker/setLocation) merged into the CMS master requiring some minor re-coding of the patch, but before the effort gets put into that, is there support for the Nicholas Dionysopoulos's proposed solution, at least as a workaround before the installer can be written to remove the root of the problem?

See the CMS Issue tracker for further details.

http://joomlacode.org/gf/project/joomla/tracker/?action=TrackerItemEdit&tracker_item_id=25663

**Comment by pasamio** [comment]
Is there a pull request for the issue?

**Comment by pjwiseman** [comment]
This issue is being resolved by moving the installer code from platform to cms.  This move was agreed by platform.  This agreement allowed the code change to proceed immediately in CMS 2.5.x.  Testing of the code move has commenced under CMS tracker #29235.

Platform discussion:
platform (https://groups.google.com/group/joomla-dev-platform/browse_thread/thread/1d6727a66eca9927)

CMS Tracker for the code move:
http://joomlacode.org/gf/project/joomla/tracker/?action=TrackerItemEdit&tracker_item_id=29235

A separate platform pull for the code move will be performed once CMS testing is complete.
